meta/documents/changelog_en.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes for IO
 
+## v5.0.10 (2020-08-27) <a href="https://github.com/plentymarkets/plugin-io/compare/5.0.9...5.0.10" target="_blank" rel="noopener"><b>Overview of all changes</b></a>
+
+### Fixed
+
+- URLs with more than 6 segments are correctly recognized and displayed as 404 pages.
```

Comment:
URLs with more than 6 segments are now correctly recognized and displayed as 404 pages.